src/assets/persons.js

```diff
@@ -17093,6 +17093,24 @@ export const data = {
         country: "Canada",
       },
     },
+
+    ,{
```

Comment:
@atrieceeph289 there are two `,` after `{`. this is causing [Netlify CD](https://app.netlify.com/sites/alumni-board/deploys/5f74cf9d9596a000079fee3f) to fail. Please fix it so that this PR can be merged.